### PR TITLE
MGMT-4865 - Resolve installation disk symlink before running coreos-installer

### PR DESCRIPTION
**Pull request by omertuc**
Overcome https://github.com/coreos/coreos-installer/issues/512 bug.

coreos-installer has a bug where when a disk has busy partitions, it will
print a confusing error message if that disk doesn't have a `/dev/*` style path.
The service may give us paths that don't have the `/dev/*` path format but instead
are symlinks to the actual `/dev/*` path. e.g. `/dev/disk/by-id/wwn-*`.
To fix the bug we simply resolve the symlink and pass the resolved link to coreos-installer.

### CODE REVIEW

**Comment by omertuc** [comment]
Force pushed to update irrelevant/wrong commit message

File: src/ops/ops.go [hunk 1]
```diff
@@ -174,9 +174,24 @@ func (o *ops) SystemctlAction(action string, args ...string) error {
...
 func (o *ops) WriteImageToDisk(ignitionPath string, device string, progressReporter inventory_client.InventoryClient, extraArgs []string) error {
+	// Overcome https://github.com/coreos/coreos-installer/issues/512 bug.
+	// coreos-installer has a bug where when a disk has busy partitions, it will
+	// print a confusing error message if that disk doesn't have a `/dev/*` style path.
+	// The service may give us paths that don't have the `/dev/*` path format but instead
+	// are symlinks to the actual `/dev/*` path. e.g. `/dev/disk/by-id/wwn-*`.
+	// To fix the bug we simply resolve the symlink and pass the resolved link to coreos-installer.
+	linkTarget, err := filepath.EvalSymlinks(device)
+	if err != nil {
+		o.log.Warnf("Failed to os.ReadLink(%s): %s. Continuing with %s anyway.",
```
Comment (michaellevy101) [inline src/ops/ops.go:185]: Please change the message to: Failed to EvalSymlinks...

**Pull request merged**